funnel/util.py

```diff
@@ -17,3 +18,25 @@ def geonameid_from_location(text):
         geonameids = [field['geoname']['geonameid'] for field in response['result'] if 'geoname' in field]
         return set(geonameids)
     return None
+
+
+def format_twitter_handle(handle):
+    """
+    formats a user given twitter handle
+    Eg:  -> shreyas_satish, @shreyas_satish -> shreyas_satish
+    Returns None for invalid cases.
+    Twitter restricts the length of handles to 15. 16 is the threshold here, since
+    a user might prefix their handle with an '@', a valid case.
+    Tested in tests/test_util.py
```

Comment:
Docstrings should be in ReStructuredText. This one will format badly in ReST. (We don't generate documentation from docstrings in Funnel, but only because we've limited it to libraries so far (Coaster and Flask-Lastuser).)